contracts/XVSVault/TimeManagerV5.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.5.16;
+
+contract TimeManagerV5 {
```

Comment:
Why are we creating this TimeManagerV5 instead of using it from `solidity-utilities`?.